setup.cfg

```diff
@@ -1,3 +1,6 @@
+[flake8]
+max-line-length=100
```

Comment:
Would you mind changing that to 80? I usually work with two or three editors one next to each other and 80 is the perfect size for that.